src/host/selectionState.cpp

```diff
@@ -182,7 +181,6 @@ void Selection::_RestoreDataToCursor(Cursor& cursor) noexcept
 {
     cursor.SetSize(_ulSavedCursorSize);
     cursor.SetIsVisible(_fSavedCursorVisible);
-    cursor.SetColor(_savedCursorColor);

```

Comment:
As with the alt buffer, there's no longer a need to save the cursor color here, because it's not going to be affected by the switch to a selection cursor.

Reply:
I wonder what the purpose of this code was; it suggests that there's a different cursor during selection?